<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RegisterDialog</class>
 <widget class="QDialog" name="RegisterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>423</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="dialogTitle">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="font">
      <font>
       <pointsize>14</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Регистрация</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="typeTitle">
     <property name="text">
      <string>Зарегистрироваться как:</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QComboBox" name="typeComboBox">
     <item>
      <property name="text">
       <string>Пользователь</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Разработчик</string>
      </property>
     </item>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="userBlock" native="true">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QLabel" name="userLoginTitle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Логин:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userLoginField">
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userPasswordTitle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Пароль:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userPasswordField">
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userPasswordRepeatTitle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Пароль повторно:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userPasswordRepeatField">
        <property name="maxLength">
         <number>32</number>
        </property>
        <property name="echoMode">
         <enum>QLineEdit::PasswordEchoOnEdit</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="userNameTitle">
        <property name="font">
         <font>
          <pointsize>11</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Имя:</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="userNameField">
        <property name="maxLength">
         <number>100</number>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="registerButton">
       <property name="text">
        <string>Зарегистрироваться</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Отмена</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
